frontend/src/pages/detail/components/ApplySection/ApplySection.tsx

```diff
@@ -0,0 +1,59 @@
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import { useNavigate } from 'react-router-dom';
+
+import Button from '../../../../common/components/Button/Button';
+
+function ApplySection() {
+  const navigate = useNavigate();
+  const moveToBookingPage = () => {
+    navigate('/booking');
+  };
```

Comment:
p4: 개행을 줘서 jsx와 함수 사이를 시각적으로 구분하도록 만드는 건 어때? 개인취향인 거 같아서 반영해도 좋고 안해도 괜찮을 것 같아 👍

```suggestion
  };
  
```